WordPress/Classes/ViewRelated/Post/PostEditor+MoreOptions.swift

```diff
@@ -35,32 +34,30 @@ extension PostEditor where Self: UIViewController {
             return
         }
 
-        if post.isDraft() {
-            SVProgressHUD.show(withStatus: draftStatus)
-            postService.uploadPost(post, success: { [weak self] savedPost in
-                self?.post = savedPost
-                self?.createPostRevisionBeforePreview() {
-                    completion(nil, nil)
-                }
-                SVProgressHUD.dismiss()
-                }, failure: { error in
-                    DDLogError("Error while trying to upload draft before preview: \(String(describing: error))")
-                    completion(nil, nil)
-                    SVProgressHUD.dismiss()
-            })
-        } else {
-            SVProgressHUD.show(withStatus: publishedStatus)
-            postService.autoSave(post, success: { [weak self] savedPost, previewURL in
-                self?.post = savedPost
+        SVProgressHUD.show(withStatus: publishedStatus)
```

Comment:
Yep - I think "Generating Preview..." is fine for all scenarios, since that's a good description of what we're doing.  Plus this allow us to keep the local service quite abstract and reusable from other locations.